db/migrate/20180313095604_remove_provider_options_from_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveProviderOptionsFromUsers < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :users, :provider, :string
+    remove_column :users, :uid, :string
+  end
```

Comment:
@marderer wouldn't such migration remove all `provider` and `uid` data? I was logged in once, after this migration all my logging data will be removed.

What do you think about migrating stuff to `authorizations` firstly, and only after it removes columns from `users`?

You can create rack task for that, on your like 😉

Reply:
@AleksSenkou I thought that didn't need. Because It does not delete your user, and after next sing all you dates will work and automatic created `"Authorization'` for your user